filebeat/module/mysql/slowlog/config/slowlog.yml

```diff
@@ -3,8 +3,9 @@ paths:
 {{ range $i, $path := .paths }}
  - {{$path}}
 {{ end }}
-exclude_files: [".gz$"]
+exclude_files: ['.gz$']
 multiline:
-  pattern: "^# User@Host: "
+  pattern: '^# User@Host: '
   negate: true
   match: after
+exclude_lines: ['^[\/\w\.]+, Version: .* started with:.*']   # Exclude the header
```

Comment:
I don't know, I worry it would cause more confusion than it solves. Something like "The slowlog fileset ignores the header written by MySQL on startup."?